src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Email/Blocks/email_configuration.html.twig

```diff
@@ -38,6 +38,19 @@
           <div class="js-smtp-configuration{% if emailConfigurationForm.mail_method.vars.value != smtpMailMethod %} d-none{% endif %}">
             {{ form_widget(emailConfigurationForm.smtp_config) }}
           </div>
+          {{ form_row(emailConfigurationForm.mail_type) }}
+          {{ form_row(emailConfigurationForm.log_emails) }}
+          <div class="alert alert-info" role="alert">
+            <p class="alert-text">
+              {{ 'Settings below allow you to configure DKIM signing of outgoing emails. This will greatly reduce the chance of your emails ending in SPAM folders.'|trans({}, 'Admin.Advparameters.Help') }}
+            </p>
+            <p class="alert-text">
+              {{ 'You can get the data below from your server administrator or generate it yourself by using one of freely available tools, like dkimcore.org.'|trans({}, 'Admin.Advparameters.Help') }}
```

Comment:
```suggestion
              {{ 'You can get the data below from your server administrator or generate it yourself by using one of the freely available tools, such as dkimcore.org.'|trans({}, 'Admin.Advparameters.Help') }}
```
Could we add the link to dkimcore.org such as '(DKIMCore)[https://dkimcore.org/] or is it not recommended?